docs/index.html

```diff
@@ -79,6 +79,14 @@
         <script>
             const RusfmtTagsUrl = 'https://api.github.com/repos/rust-lang/rustfmt/tags';
             const UrlHash = window.location.hash.replace(/^#/, '');
+            const queryParams = new URLSearchParams(window.location.search);
+            const versionParam = queryParams.get('version');
+            const parseVersionParam = (version) => {
+              if (version === 'master') return 'master';
+              if (version.startsWith('v')) return version;
+              return `v${version}`;
+            };
+            const versionNumber = null !== versionParam ? parseVersionParam(versionParam) : 'master';
```

Comment:
nit: maybe checking for falsyness (`!versionParam`) is better, in case the param looks like `?version=`

Reply:
hmm, semantically the user has provided a version number, an empty string in this case. so the current way clearly separates _version not provided_ case from _provided but invalid_ case. 
If its still necessary will use falsy then

Reply:
yeah but I think it's fair to assume the user didn't intend to specify a particular version since no valid version will be an empty string

Reply:
yes, an empty string may not be a valid version but what about `?version=0.0` or `?version=0n`.
do we also interpret them as having no intention to specify a version.

Reply:
IIRC URLSearchParams will return strictly `null` in the absence of the query param, so I don't want to bikeshed too much on this one. 

If anyone's interested in trying to handle the empty string version case then I'd suggest that be done in a follow up